### ISSUE TITLE
About Pretrained model and the Dataset

### ISSUE THREAD

**Issue by luoxin13** [mathilde-b/SFDA#5]
Hello, thanks for sharing the great work. I have some puzzles on this repo and 
I am wondering if it is convenient for you to get them figured out.

1. would it be possible to share the pre-trained weights of source-only and adapted target models?

2. Could you describe the data splits for site-A and site-B training and validation?
Or both of them are the same as described in #1 ?

3. As shown in the codes, the input images should be in the range [0,1],
so, did you apply normalization on each slice of images (shaped as [1, 384, 384]) 
or on the whole nii instance (shaped as [n_slices, 384, 384])?

4. Did you involve all the slices in training or filter those slices without class-1?

**Comment by mathilde-b** [comment]
Hello ! Sorry for the late reply
1/ I'll see if I can upload them yes 
2/ For the source domain, it should not have any impact, as it's supervised learning there. For the target domain, yes, it is the same as described in that question. 
3. The normalization was on each slice, but it shouldn't change too much if using the whole instance instead.
4. No, that would be cheating :) all slices are used for training. But yes, in the weakly supervised scenario, you also add the information "is the class 1 present in the slice" - this is done automatically in the loss function, no need to add something. Specifically, in the KL, the size of the prior will be passed to zero if the class is not present. For the test domain, all slices are included.
5.
